python/ray/data/read_api.py

```diff
@@ -103,23 +103,33 @@ def from_items(
     parallelism: int = -1,
     output_arrow_format: bool = True,
 ) -> MaterializedDataset:
-    """Create a dataset from a list of local Python objects.
+    """Create a :class:`~ray.data.Dataset` from a list of local Python objects.
```

Comment:
just curious. how do you verify whether the link to Dataset is correct?

Reply:
You'd have to render the documentation locally (`make develop`) or check the ReadTheDocs CI build.